_pytest/mark.py

```diff
@@ -97,6 +94,20 @@ def _for_parameterize(cls, argnames, argvalues, function):
         return argnames, parameters
 
 
+def get_empty_parameterset_mark(config, argnames, function):
+    requested_mark = config.getini('empty_parameterset')
```

Comment:
Now that I take a look another look at it, I think this option should be named `empty_parameter_set_mark` instead, what do you think?

Reply:
yup, a moment please for the fix